services/web/server/src/simcore_service_webserver/resource_manager/user_sessions.py

```diff
@@ -108,7 +108,7 @@ async def user_pressed_disconnect(self) -> None:
 
         await self._registry.set_key_alive(self._resource_key(), 1)
 
-    async def remove_socket_id(self) -> None:
+    async def remove_socket_id_after_disconnection(self) -> None:
```

Comment:
I don't why you change the naming of that function.
In effect you are moving logic naming into a deeper location.